cmd/updater/diffdumps/cmd.go

```diff
@@ -98,12 +148,18 @@ func generateK8sDiffs(outputDir string, baseZipR *zip.ReadCloser, headZipR *zip.
 			continue
 		}
 
-		// Only look at YAML files in the k8s/ folder.
-		if filepath.Dir(name) != vulndump.K8sDirName || filepath.Ext(name) != ".yaml" {
+		// Only look at YAML files in the k8s/ or istio/ folder.
```

Comment:
in case we add another for some reason, let's just say "Only look at YAML files in the given `dirName ` directory."